ads/zedo.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {validateData, loadScript} from '../3p/3p';
+
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function zedo(global, data) {
+  // check mandatory fields
+  validateData(data, ['superId', 'network', 'placementId',
+  'channel', 'publisher', 'dim'], ['charset', 'callback', 'renderer']);
```

Comment:
for better readability, can you break lines before each '[' and indent 4 spaces on the 2nd line?